config/routes.rb

```diff
@@ -9,9 +9,11 @@
   # root "articles#index"
   get "users", to: "users#index"
   get "/users/:id", to: "users#show"
-  get "posts", to: "posts#index"
-  get "/posts/:id", to: "posts#show"
   get "/users/:id/posts", to: "users#post"
   get "/users/:id/posts/:idp", to: "users#detail"
+  get "posts", to: "posts#index"
+  get "/posts/:id", to: "posts#show"
+  
+  resources :comments
```

Comment:
- [ ] Please add  routes for creating a new post and when you run rails routes you should have these routes:
```ruby
GET   /posts/new                   posts#new
POST   /posts(.:format)                     posts#create
```

- [ ] Please add a route for creating a new like and when you run rails routes you should have this route:
```ruby
POST /posts/:post_id/likes(.:format) likes#create
```
[Hint]: It would be better professionally If you replaced all routes and make it like this:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, :new, :create] do
      resources :comments, only: [:create, :new]
      resources :likes, only: [:create]
    end
  end
 
  root to: 'users#index'
```

Reply:
done
![image](https://user-images.githubusercontent.com/13853450/192126630-06409e9f-315b-4003-84c4-0f07c0cadd02.png)

Reply:
Done - Add new post.
![image](https://user-images.githubusercontent.com/13853450/192126773-3521a915-bf7c-43f9-afb4-a5e27d4d8e9e.png)